cmd/beast/create_admin_author.go

```diff
@@ -86,3 +94,139 @@ var createAdminCmd = &cobra.Command{
 		utils.CreateAdminOrAuthor(Name, Username, Email, PublicKeyPath, Password, "admin")
 	},
 }
+
+var createMultipleAuthorCmd = &cobra.Command{
+	Use:   "create-multiple-author",
+	Short: "Creates multiple new authors",
+	Long:  "Creates multiple new authors using given .csv file",
+	PreRun: func(cmd *cobra.Command, args []string) {
+		if CsvFile == "" {
+			fmt.Printf("Csv File path not provided")
+			os.Exit(1)
+		}
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		csvFile := filepath.Join(BEAST_GLOBAL_DIR, CsvFile)
+
+		config.InitConfig()
+
+		auth.Init(core.ITERATIONS, core.HASH_LENGTH, core.TIMEPERIOD, core.ISSUER, config.Cfg.JWTSecret, []string{core.USER_ROLES["author"]}, []string{core.USER_ROLES["admin"]}, []string{core.USER_ROLES["contestant"]})
```

Comment:
What does this do?

Reply:
InitConfig loads the config from the global config file
auth.Init initializes the constants in pkg/auth.go from contants.go